package.json

```diff
@@ -33,10 +33,11 @@
     "build:svgs": "lerna run svgs --scope bpk-svgs",
     "build": "npm run build:tokens && npm run build:svgs && lerna run build && (cd ./native && npm run link)",
     "storybook:dist": "build-storybook -c .storybook -o dist/storybook",
+    "ios-api:dist": "git submodule update --init --recursive && (cd backpack-ios && bundle install && npm i && (cd Example && bundle exec pod install) && gulp && rake \"docs['../dist/ios/']\")",
```

Comment:
- Potentially should move `git submodule update --init --recursive` to a new command
 - Should probably be called by `docs:dist` instead of here (as then it will be called once for both android+ios, instead of once for each